packages/cf-style-const/src/variables.js

```diff
@@ -4,6 +4,13 @@ const colorOffsetLight = 0.15;
 const colorOffsetDark = 0.07;
 
 export default {
+  breakpoints: {
+    mobile: '@media (min-width: 13.6em)',
+    mobileWide: '@media (min-width: 30.4em)',
```

Comment:
it's unmaintainable to support 5 media breakpoints, in my experience it's a lot of effort to work with 2 points. Where this list come from? We should have a conversation with design team and agreed what we support.